### DISCUSSION TITLE
Large difference in ground temperature between the CLM5 and CLM 4.5 physics over the permafrost region.

### DISCUSSION THREAD

**Discussion by AdrienDams** [ESCOMP/CTSM#1960] [Q&A] [answered]
Heidrun Matthes and I have been working on a CTSM evaluation study over the Arctic for the past year. At the last Land Working Group meeting, I have shown that we observed a strong cold bias using CTSM compared to a series of in-situ and satellites observations.

 Recently, we compare two runs using the same set-up (see below) but with two different physics: the default one with CLM5, and another one with CLM4.5 physics (by changing the compset). 

The difference between these two runs shows that the CLM5 run is significantly colder over Siberia and the Eastern Canadian Peninsula than CLM4.5. The two figures below show the difference in ground temperature at 1m between the two runs in (Fig. 1) January and (Fig. 2) July for the mean period 1980-2021. The blue colours mean that the default run (CLM5) is colder.

![TSOI 01 diff 57_DOM02_001-57_DOM02_043](https://user-images.githubusercontent.com/24732655/223085278-37f2708b-36b2-4491-9503-cd5bfc1f647a.png)
![TSOI 07 diff 57_DOM02_001-57_DOM02_043](https://user-images.githubusercontent.com/24732655/223085338-1cbf8bd5-cabc-4951-9e2e-e31ee95cdca6.png)

At first we thought that it was a difference in the snow representation as it was significantly improved in CLM5. However, the snow representation between these two runs is very similar (fig. 3 is January only).  Blue colours mean the CLM5 snow depth is lower.

![H2OSNO 01 diff 57_DOM02_001-57_DOM02_043](https://user-images.githubusercontent.com/24732655/223085763-80294c70-d735-48b8-928d-67ee66d8875f.png)

The most significant difference we found between those two runs is in their soil water content, especially the soil ice below (fig. 4 is January only). I can also provide volumetric soil water and soil liquid water if needed. Blue colours mean the CLM5 soil ice content is lower.

![SOILICE 01 diff 57_DOM02_001-57_DOM02_043](https://user-images.githubusercontent.com/24732655/223087779-b6ff374d-d51f-4206-9e55-e5307c422701.png)

Our question is simply how can we explain this difference? Have any of you encountered this? Do you have any recommendations for us to resolve this issue?

Model set-up:
- CTSM version: 5.1.dev086
- Compset: 2000_DATM%GSWP3v1_CLM??%SP_SICE_SOCN_SROF_SGLC_SWAV
- Domain: Arctic domain above 57°N, Icosahedral grid (240860 pixels with resolution = 12 km²)
- Atmospheric forcings: ERA5 from 1980-2021 after 30 year spin-up
- Modified parameters:
  - reset_snow = .true.
  - h2osno_max = 800

**Comment by wwieder** [comment]
Hi Adrien, 

Thanks for reaching out.
I'd suspect these changes come from updates to snow density to CLM5 that reduced the insulation from snow, thereby making arctic soils colder.  Dave showed similar results in the [CLM5 overview paper in JAMES (2019)](https://agupubs.onlinelibrary.wiley.com/doi/full/10.1029/2018MS001583).   From this paper there's a link to the [diagnostics package results](https://www2.cesm.ucar.edu/experiments/cesm2.0/land/diagnostics/clm_diag_PCKG.html) (there are many), which includes a similar [CLM4.5 vs. 5-SP simulations](https://webext.cgd.ucar.edu/I20TR/clm50sp_clm50dev001_1deg_GSWP3V1_hist/lnd/clm50sp_clm50dev001_1deg_GSWP3V1_hist.1995_2014-clm45sp_clm50d001_1deg_GSWP3V1_hist.1995_2014/setsIndex.html) with much colder arctic coil temperatures.  

I'm not that familiar with this part of the model, but reading Dave's paper it looks like the [changes in snow density we're using in CLM5 are described here ](https://agupubs.onlinelibrary.wiley.com/doi/10.1002/2017MS000988)

Hope that helps.
Will

**Reply by AdrienDams** [reply]
Thanks for your quick answer Will!

These diagnostic packages look very very valuable to us, I didn't know they existed before. However, I am not sure how to interpret them.

Here is the ground temperature difference at 1m. The colour scale goes to -15.21 (I presume it's the maximum over the whole region) but it is difficult to know how large is the difference over Siberia. Can we access any netcdf from this data or do you know more about those?

![image](https://user-images.githubusercontent.com/24732655/223146767-9b94625a-8583-4068-9b69-b9fa9ec2ffd3.png)

**Reply by wwieder** [reply]
Yes, that -15.21 value is the global maximum difference between the simulations.
If you have access to cheyenne the files are on disk here `/glade/p/cgd/tss/people/oleson/CLM_LAND_ONLY_RELEASE`. I think the organization of the files is pretty self evident, but reach out if you're confused the particular cases you may be interested in.

Otherwise, see the data availability section of Dave's paper.  I think we also made these data available on the climate data gateway.

**Comment by dlawrenncar** [comment]
I agree with Will that I anticipate that the fresh snow density changes are
the most likely source of the difference.  I'm not sure what to make of the
soil ice difference, though I think you need to be careful comparing ice
water content at different levels between CLM5 and CLM4.5 because the soil
layers changed.

There have been several conversations and paper threads that suggest that
perhaps the density is too high now, which is leading to less snow
insulation and cooler temperatures.  My theory, supported somewhat by this
paper <https://tc.copernicus.org/articles/16/4201/2022/>, is that it is
possible that the bulk density is actually ok, but that the density profile
is incorrect.  In particular, since CLM does not represent depth hoar,
which is common across much of the domain as far as I understand.  If there
was a depth hoar layer within the snow pack, that could really increase the
insulation, but with perhaps not that much impact on the snow full column
density.  That paper uses essentially a scaling factor to correct for this
problem, which is unsatisfying.  To resolve this correctly, I guess we
would need a more advanced snow model that captures depth hoar, which as
far as I am aware does not exist in the context of a global model.

To resolve your situation at this point, my only suggestion is to (a) play
with the snow parameters to try to reduce the densification process or (b)
revert to CLM4.5 snow physics, which might produce overall poorer snow
density, but which might have better 'effective' density.  If you do try
option (b), I would be curious to see what happens with the soil
temperatures, especially to try to confirm that there isn't any other
reason why the soil temperature differs between model versions.

**Reply by AdrienDams** [reply]
Thank you so much for sharing Victoria's paper. Do you have a list of all recent papers using CLM5? I am aware of this one, but it only goes up to 2020: https://www2.cesm.ucar.edu/models/cesm2/land/clm_bibliography.htm

**Reply by olyson** [reply]
I've updated the bibliographies linked on this page:

https://www.cesm.ucar.edu/models/clm/publications

The 2020 list is now 2022.
There is some automated method that updates the "Last Update" note that isn't working. Ryan will be moving these pages to Drupal soon and will look at fixing this.

**Comment by wwieder** [comment]
@AdrienDams I'm curious if you have any new insights on this discussion?  Specifically, as we think about calibrating CLM for CESM3 are there datasets to compare with, metrics to be considering, or anything to help us think about snow - soil temperature dynamics? Thanks in advance for your perspective on this.

**Reply by AdrienDams** [reply]
@wwieder, indeed, we do! In a turn of events, we shifted the focus of my thesis project to address what we believe is a systematic cold bias over permafrost areas in many LSMs including JSBACH, CLASS, ISBA, Noah-MP, CLM5, and partially JULES.

We evaluated CTSM soil temperature wih the same domain with 2 datasets: a home-made dataset of 278 boreholes over the Arctic and remote sensing products from [ESA-CCI](https://climate.esa.int/en/projects/permafrost/). All the scripts I used are here: https://github.com/AdrienDams/cegio.

Our evaluation revealed that, in the control run, soil temperatures are consistently underestimated across most of the permafrost region, except for certain high-altitude areas (see Figure 1, on the left), and this discrepancy was observed for all seasons and depths (see Figure 2). Figure 1 is the period (1997 to 2019) MAGT at 1m depth difference between CLM5 and ESA-CCI. Dark blue means CLM5 soil temperature is colder than ESA-CCI. ESA-CCI data is extrapolated on the CLM5 grid.

![TSOI_control_sturm](https://github.com/ESCOMP/CTSM/assets/24732655/164c4d0c-1ad6-402d-a031-aeb5a3f39592)

and Figure 2 is the period averaged (1980-2021) of monthly soil temperature at 4 different depths (-20, -80, -160 and -320 cm) for the observations (black) and control run (blue), all values come from an average of the 278 stations through the full period:

![annual_cycle 57_DOM02_001-57_DOM02_051](https://github.com/ESCOMP/CTSM/assets/24732655/313157d6-9c32-4143-970f-d727b855424b)

What you see in the second column of figure 1 and the green line in figure 2 is a sensitivity experiment where we expanded Victoria study to our entire domain using the [Sturm scheme](https://doi.org/10.3189/S0022143000002781) (I can't stress enough how great this Sturm paper is). 

By applying this scheme, we believe that the cold bias has been significantly reduced, especially during the winter months, as anticipated. However, it's worth noting that we now encounter a strong warm bias in high-altitude areas. We suspect that this bias was already present in the control run (probably due to the higher resolution atmospheric forcings from ESA-CCI) and has been only amplified. Moreover, Victoria and Nick expected Sturm to not work properly over alpine snowpack.

Interestingly, the ALT remains consistant as it depends more on MaxAGT than on MAGT:

![ALT_control_sturm](https://github.com/ESCOMP/CTSM/assets/24732655/1601219c-1676-464b-ad66-5191ea9dfdce)

I want to stress that this are very preliminary results and conclusions that only Heidrun and I have seen. I haven't share it with Victoria or Nick Rutter yet.

Lastly, I briefly want to mention that we conducted another sensitivity experiment where we derived soil texture and SOM using an alternative method used in ESA-CCI. However, this aspect is still a work in progress, although it shows promising results in addressing the cold bias during the summer months.

**Reply by wwieder** [reply]
@AdrienDams this is really encouraging, thanks for sharing your results.  I assume that you're working on writing this up with Heidrun?  I want to emphasize that this should be your focus, but I also wondered:  
- If you would be willing to share your results at an upcoming CLM meeting?  We typically meet Thursday at 1:00 Mountain, but can find a more convenient time / date if that's better for you.
- Do you have an implementation of the Sturm scheme that we could test out for broader adoption in CLM? As background, we're having some issues with [vegetation growth and survival](https://github.com/ESCOMP/CTSM/discussions/2011) in parts of the Arctic and I wonder if these changes to soil temperatures may help plants survive better in some of these cold regions?

Thanks again for this update.

**Reply by AdrienDams** [reply]
@wwieder Thanks! I hope to finish my thesis by the end of November, and then I'll ask Vicky and Nick to write a paper on the Sturm part
- Of course I will be happy to share my results and the time is not a problem. However, I am busy on Sep 14. Is it possible on Sep 21?
- You can find how I have added the Sturm scheme in CLM [here](https://github.com/ESCOMP/CTSM/commit/f21a759853a611ee938c9b2ce06b283e87e5cf61) (and I made a mistake that I corrected [here](https://github.com/ESCOMP/CTSM/commit/2a249aff109028bffdff88e1cb074db775653e4e)). This was initially done by Vicky and Leanne Wake.
- Regarding vegetation, well we are using a SP compset so I can't give your a concrete answer from our runs, and we are not very familiar how plant survival works. But if extreme cold temperature in winter are killing your plants, it might help

**Reply by wwieder** [reply]
Thanks for providing this code @AdrienDams. Assuming it's an option we want to provide in the code maybe you (or Vicky) can open a PR to do this formally?  That way our github repo reflects your contributions to the code.

In the meantime, we'd like to test the impacts of this on soil temperatures more widely if that's OK?

**Reply by AdrienDams** [reply]
Sounds good! I am really eager to contribute.

**Comment by wwieder** [comment]
Of note here Daniele Peano at CMCC noted we're seeing a "_cold bias in the North Hemisphere high-latitude during our coupled 2000 simulations_" and decided to revert the snow density options to the CLM4.5 ones:

```
lotmp_snowdensity_method = 'TruncatedAnderson1976'
wind_dependent_snow_density = .false.
albice=0.6,0.4
```

**Comment by AdrienDams** [comment, marked as answer]
If you are interested to know more about this, our paper on the use of the Sturm scheme in CLM has been finaly published!

https://tc.copernicus.org/articles/19/1539/2025/

**Reply by wwieder** [reply]
excellent!  Thanks for this heads up.  Hope you're doing we'll @AdrienDams

**Reply by AdrienDams** [reply]
Doing great! I sent you an e-mail by the way, maybe it’s your spam folder
